dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html

```diff
@@ -1,72 +1,346 @@
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        height: 100vh;
+        margin: 0;
+      }
+
+      .wrapper {
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        gap: 10px;
+        width: 20%;
+      }
+
+      .nav-right {
+        margin-top: 10px;
+        display: flex;
+        align-items: start;
+        gap: 10px;
+        justify-content: start;
+      }
+
+      .nav-right img {
+        max-height: 64px;
+        border: 1px solid white;
+        width: 100px;
+      }
+
+      .right-content {
+        display: flex;
+        height: 10px;
+        flex-direction: column;
+        gap: 10px;
+      }
+
+      .right-content h3{
+        margin: 0px;
+      }
+
+      .right-content p{
+        margin: 0px;
+      }
+
+      .notification-container {
+        background: #f5fbff;
+        display: flex;
+        gap: 8px;
+        width: 100%;
+        padding: 2rem;
+        align-items: start;
+        justify-content: center;
+        border-radius: 5px;
+        box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+      }
+
+      .notification-content {
+        font-size: 14px;
+        display: flex;
+        flex-direction: column;
+        gap: 10px;
+        line-height: 20px;
+      }
+
+      .fa-circle-info {
+        color: #093371;
+        font-size: 20px;
+      }
+
+      .login-container {
+        background-color: white;
+        padding: 2rem;
+        border-radius: 5px;
+        box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+        width: 100%;
+      }
+
+      .login-container h1 {
+        margin-bottom: 12px;
+        font-size: 1rem;
+        color: #333;
+      }
+
+      .login-container label {
+        font-size: 0.9rem;
+        color: #333;
+        display: block;
+        text-align: left;
+      }
+
+      form {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+      }
+
+      form div {
+        width: 100%;
+      }
+
+      input {
+        width: 93%;
+        padding: 10px;
+        margin-bottom: 10px;
+        border-radius: 5px;
+        border: 1px solid #ccc;
+      }
+
+      .login-container input[type="submit"] {
+        background-color: #1565c0;
+        color: white;
+        padding: 10px;
+        border: none;
+        border-radius: 5px;
+        width: 100%;
+        font-size: 1rem;
+        cursor: pointer;
+        transition: background-color 0.3s ease;
+      }
+
+      .login-container input[type="submit"]:hover {
+        background-color: #004a8b;
+      }
+
+      .logo-container {
+        color: white;
+        width: 100%;
+      }
+
+      .nav {
+        margin: 0px 20px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+      }
+
+      p {
+        color: #a0adba;
+      }
+
+      .footer {
+        position: fixed;
```

Comment:
I guess because this is fixed, you get a slightly weird effect if you resize the window to be very small where the footer bar appears above the login box:

<img width="495" alt="image" src="https://github.com/user-attachments/assets/0d90b52e-5864-4d28-87fb-68616cf89895">

I think it's fine, though; we can just let people size their windows appropriately if they're on this page 😄

Reply:
Didn’t catch this as I'm only able to test/preview after building the containers.